fabric-server-consent-api-v1/src/main/java/net/fabricmc/fabric/api/server/consent/v1/FabricServerConsent.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.server.consent.v1;
+
+import java.util.Collections;
+import java.util.List;
+
+import net.fabricmc.fabric.impl.server.consent.FabricServerConsentImpl;
+
+public final class FabricServerConsent {
+	/**
+	 * Returns whether the Fabric Server Consent API is enabled on this server.
+	 *
+	 * @return {@code true} if it is enabled, {@code false} otherwise
+	 */
+	public static boolean isEnabled() {
+		return FabricServerConsentImpl.enabled;
+	}
+
+	/**
+	 * The list of illegal mods as specified by the consents file.
+	 *
+	 * @return the list of illegal mods
+	 */
+	public static List<String> getIllegalMods() {
+		return Collections.unmodifiableList(FabricServerConsentImpl.illegalMods);
+	}
+
+	/**
+	 * The list of illegal features as specified by the consents file.
+	 *
+	 * @return the list of illegal features
+	 */
+	public static List<String> getIllegalFeatures() {
```

Comment:
`public static List<Identifier> getFlags()`

I dont think features is quite right, flags might be better?

Reply:
Maybe "mod flags"? Since 1.19.4 we have "feature flags"

Reply:
I have changed it to `flags` for now.